src/components/DrawingPage/DrawingPage.js

```diff
@@ -22,6 +25,10 @@ export default class DrawingPage extends Component {
             .then(res => {
                 this.context.getPrompt(res.prompt)
             })
+        DigiDoodleApiService.getAllPlayersInGame(this.context.gameId)
+            .then(res =>
+                this.context.setPlayers(res[0].username, res[0].score),
```

Comment:
Instead of hard coding this to only work for one player, you should write it so it doesn't need to be refactored when we have multiple players in a game. Rewrite this and the setPlayers function to use all the players returned by 'getAllPlayersInGame'